components/gitpod-db/src/typeorm/entity/db-workspace-cluster.ts

```diff
@@ -88,13 +88,13 @@ export class DBWorkspaceCluster implements WorkspaceCluster {
     })
     admissionConstraints?: AdmissionConstraint[];
 
-    @PrimaryColumn({
+    @Column({
         type: "varchar",
         length: 60,
     })
     applicationCluster: string;
 
-    @PrimaryColumn({
```

Comment:
In case anyone wonders why this field was marked as primary but actually wasn't part of the primary key. This information is only used when using the `synchronize` feature of TypeOrm which we don't, so it can easily go out of sync 😬.